src/interface/desktop/main.js

```diff
@@ -428,162 +428,211 @@ function addCSPHeaderToSession () {
+         });
+     } else {
+         win.once('ready-to-show', () => { win.show(); });
+     }
+ };
+
+ const createShortcutWindow = (tab = 'index.html') => {
+     var shortcutWin = new BrowserWindow({
+         width: 400,
+         height: 600,
+         show: false,
+         titleBarStyle: titleBarStyle,
+         autoHideMenuBar: true,
+         frame: false,
+         webPreferences: {
+             preload: path.join(__dirname, 'preload.js'),
+             nodeIntegration: true,
+         }
+     });
+     shortcutWin.setMenuBarVisibility(false);
+     shortcutWin.setResizable(false);
+     shortcutWin.setOpacity(0.95);
+     shortcutWin.setBackgroundColor('#f5f4f3');
+     shortcutWin.setHasShadow(true);
+     shortcutWin.setVibrancy('popover');
+
+     shortcutWin.loadFile(tab);
+     shortcutWin.once('ready-to-show', () => {
+         shortcutWin.show();
+     });
+
+    shortcutWin.on('closed', () => {
+        shortcutWin = null;
     });
 
-    job.start();
-
-    win.loadFile(tab)
-
-    if (firstRun === true) {
-        firstRun = false;
-
-        // Create splash screen
-        let splash = new BrowserWindow({width: 400, height: 400, transparent: true, frame: false, alwaysOnTop: true});
-        splash.setOpacity(1.0);
-        splash.setBackgroundColor('#d16b4e');
-        splash.loadFile('splash.html');
-
-        // Show splash screen on app load
-        win.once('ready-to-show', () => {
-            setTimeout(function(){ splash.close(); win.show(); }, 4500);
+     return shortcutWin;
+ };
+ app.whenReady().then(() => {
+     addCSPHeaderToSession();
+
+     ipcMain.on('set-title', handleSetTitle);
+     ipcMain.handle('handleFileOpen', (event, type) => handleFileOpen(type));
+     ipcMain.on('update-state', (event, arg) => { console.log(arg); event.reply('update-state', arg); });
+     ipcMain.on('needsSubscription', (event, arg) => { console.log(arg); event.reply('needsSubscription', arg); });
+     ipcMain.on('navigate', (event, page) => { win.loadFile(page); });
+     ipcMain.on('navigateToWebApp', (event, page) => { shell.openExternal(`${store.get('hostURL')}/${page}`); });
+
+     ipcMain.handle('getFiles', getFiles);
+     ipcMain.handle('getFolders', getFolders);
+     ipcMain.handle('removeFile', removeFile);
+     ipcMain.handle('removeFolder', removeFolder);
+     ipcMain.handle('setURL', setURL);
+     ipcMain.handle('getURL', getURL);
+     ipcMain.handle('setToken', setToken);
+     ipcMain.handle('getToken', getToken);
+     ipcMain.handle('getUserInfo', getUserInfo);
+     ipcMain.handle('syncData', (event, regenerate) => syncData(regenerate));
+     ipcMain.handle('deleteAllFiles', deleteAllFiles);
+
+     createWindow();
+
+     app.setAboutPanelOptions({
+         applicationName: "Khoj",
+         applicationVersion: khojPackage.version,
+         version: khojPackage.version,
+         authors: "Saba Imran, Debanjum Singh Solanky and contributors",
```

Comment:
Let's change this to Khoj AI!

```suggestion
         authors: "Khoj AI",
```